### ISSUE TITLE
Update script

### ISSUE THREAD

**Issue by ybjlus** [pablobiedma/Ticket-SwapMe#17]
Hi @pablobiedma 

Love the work you've put in to this, but I think some elements are not working anymore. I get the "unable to locate element" error for example.

Would it be possible to update the script so it works with the latest changes to the site?

Would love to help out where I can btw!

**Comment by pablobiedma** [comment]
Hello @ybjlus thank you for your interest in this repository :) 

I understand that it is no longer working as intended. 

One of the main reasons is that I originally based it on XPATHs that are dynamically changing so the code is not finding the right buttons as the links are different. 

At present I am extremely busy with work, study, and projects, and, much to my dismay I wouldn't be able to devote the required time to get this to work 100% again. I really hope you understand. 

Nonetheless, it would be amazing if you could make some functioning modifications, for instance, updating the XPaths, or changing the structure of the script to make it generalizable rather than XPath-specific and requiring constant updates (maybe GPT can help in modifying the code). If you managed to do this, I would be most thankful, feel free to create a Pull Request and I'll accept it. 

Thank you for your understanding, I hope you understand. Keep me updated.

All the best,
Pablo

**Comment by ybjlus** [comment]
Of course I understand! Its a hobby project after all!

Yeah I saw the xpath's being wrong, I've corrected a few and it definitely solves some of the problems.  But after some research I think the api approach is more viable and more future proof. Especially for tickets which are like really really popular. 

Thnx for the reply btw!

**Comment by mathieu20152** [comment]
Is it possible to share the updated code? Would love to get some tickets for me and my friends.

**Comment by GerdJan05** [comment]
Hi ybjlus, did you share it anywhere?

Thanks in advance!

**Comment by ybjlus** [comment]
> Hi ybjlus, did you share it anywhere?
> 
> Thanks in advance!

Hi,

No I didn't, because I just played around with it. Quickly corrected some x paths to see how it worked.

**Comment by GerdJan05** [comment]
Hi,

I think you won't share it then, or do you maybe want to share it with me?

I'm trying to get tickets for my friends, but this site is the worst.

**Comment by ybjlus** [comment]
> Hi,
> 
> I think you won't share it then, or do you maybe want to share it with me?
> 
> I'm trying to get tickets for my friends, but this site is the worst.

I didn't save anything, it was just a quick test to see how this script worked.

**Comment by Purrloin4** [comment]
Ticketswap also updated their anti botting protection, you will just get blocked by a 'verify your human' thing

**Comment by Williehack** [comment]
so this doesn't work anymore?

**Comment by Purrloin4** [comment]
@Williehack Nope, am currently trying a macro on my android using MacroDroid.

**Comment by Williehack** [comment]
okay how can i uninstall the npm (if that makes sense, i dont know anything about this I just wanted to reserve tickets on ticketswap lol)

**Comment by Purrloin4** [comment]
@Williehack uninstall the what?
